### PR TITLE
Add Capability Levels to OperatorHub items

### PR DESCRIPTION
**Pull request by rhamilto**
Resolves https://issues.redhat.com/browse/CONSOLE-1972

Note I have not added the help popover as it will not work with the existing modal, but the changes in #3757 should allow for its addition.

Screen Shot 2020-01-08 at 10 11 33 AM
Screen Shot 2020-01-08 at 10 11 42 AM
Screen Shot 2020-01-08 at 10 11 53 AM

### CODE REVIEW

**Comment by rhamilto** [comment]
cc:  @jeff-phillips-18

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx [hunk 1]
```diff
@@ -1,18 +1,61 @@
...
 
+const CapabilityLevel: React.FC<CapabilityLevelProps> = ({ capabilityLevel }) => {
+  const levels = [
+    'Basic Install',
+    'Seamless Upgrades',
+    'Full Lifecycle',
+    'Deep Insights',
+    'Auto Pilot',
+  ];
+  const capabilityLevelIndex = _.indexOf(levels, capabilityLevel);
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx]: nit: no need for lodash here

```suggestion
  const capabilityLevelIndex = levels.indexOf(capabilityLevel);
```

**Pull request merged**